sdks/python/container/run_validatescontainer.sh

```diff
@@ -24,10 +24,13 @@
 # REGION       -> Region name to use for Dataflow
 #
 # Execute from the root of the repository:
-#     test Python3.8 container:
+#     test Python3.8 x86 container:
 #         ./gradlew :sdks:python:test-suites:dataflow:py38:validatesContainer
-#     or test all supported python versions together:
+#     or test all supported python versions x86 containers together:
 #         ./gradlew :sdks:python:test-suites:dataflow:validatesContainer
+#
+# Note: ARM test suites only run on github actions. For example, to test Python3.8 ARM containers,
```

Comment:
I would make platform an argument to this script, or at least set the environment variable in gradle, instead of in GH actions. That would make it easier to trigger these tests locally, and would make it easier for us to pivot to test on Jenkins if we don't sort out pushing to GCR from GH actions in time.

Reply:
Added.